source/features/restore-file.tsx

```diff
@@ -53,27 +62,34 @@ async function restoreFile(progress: (message: string) => void, menuItem: Elemen
 		throw new Error('Restore failed: File too big');
 	}
 
-	let {pathname} = menuItem.previousElementSibling as HTMLAnchorElement;
-	// Check if file was deleted by PR
-	if (menuItem.closest('[data-file-deleted="true"]')) {
-		progress('Undeleting…');
-		const [nameWithOwner, headBranch] = select('.head-ref')!.title.split(':');
-		pathname = `/${nameWithOwner}/new/${headBranch}?filename=${filePath}`;
-	} else {
-		progress('Committing…');
-	}
+	const [nameWithOwner, prBranch] = select('.head-ref')!.title.split(':');
+	progress(menuItem.closest('[data-file-deleted="true"]') ? 'Undeleting…' : 'Committing…');
```

Comment:
A little confused why it's called "Undelete" instead of "Restore", "Revert", "Undo" or whatever

Reply:
It sounds funny :P

But otherwise maybe restore is more in line with the feature name...